packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -283,10 +284,6 @@ class SortedTable extends Component {
     userData: PropTypes.any,
   }
 
-  static defaultProps = {
-    itemsPerPage: 10,
```

Comment:
Add an optional `defaultItemsPerPage` prop so that we can set the home page to 20. 10 is too little IMO. And the actual number of items per page would fall back like this: cookie > `props.defaultItemsPerPage` > `DEFAULT_ITEMS_PER_PAGE`.